README.md

```diff
@@ -1,2 +1,193 @@
-# Hellow-World
-My fist module one task
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Hello World Project</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello World]** is a project to understand the basics of Github flow and lint rules
+
+## 🛠 Built With <a name="HTML/CSS"></a>
+
+### Tech Stack <a name="HTML/CSS"></a>
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[index.html file]**
+- **[style.css file]**
+- **[.gitignore file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+### Install
+
+Install this project with:
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
```

Comment:
- [ ] Kindly add the step on how to run your project locally. That will make your README looks more professional and help other developers to know how to run you project.